src/utilities/params.ts

```diff
@@ -81,7 +106,7 @@ export function getParamValue<T extends Param>(value: string, param: T): unknown
   return value
 }
 
-export function setParamValue(value: unknown, param: Param): string {
+export function setParamValue(value: unknown, param: Param): string | undefined {
```

Comment:
Seeing the `undefined` show up here makes more sense to me. Is the thought that when we're setting all `undefined` values will be coerced to empty strings?

Reply:
I reverted this change along side `getParamValue`, I think they should mirror each other